packages/cashier/src/constants/routes-config.js

```diff
@@ -8,81 +8,103 @@ import { AccountTransfer, Deposit, OnRamp, P2PCashier, PaymentAgent, PaymentAgen
+                },
+                {
+                    path: routes.cashier_pa,
+                    component: PaymentAgent,
+                    getTitle: () => localize('Payment agents'),
+                    icon_component: 'IcPaymentAgent',
+                    is_affiliate: false,
+                },
+                {
+                    path: routes.cashier_acc_transfer,
+                    component: AccountTransfer,
+                    getTitle: () => localize('Transfer'),
+                    icon_component: 'IcAccountTransfer',
+                    is_affiliate: true,
+                },
+                {
+                    path: routes.cashier_pa_transfer,
+                    component: PaymentAgentTransfer,
+                    getTitle: () => localize('Transfer to client'),
+                    icon_component: 'IcAccountTransfer',
+                    is_affiliate: false,
+                },
+                {
+                    path: routes.cashier_p2p,
+                    component: P2PCashier,
+                    getTitle: () => localize('Deriv P2P'),
+                    icon_component: 'IcDp2p',
+                    is_affiliate: false,
+                },
+                {
+                    path: routes.cashier_p2p_verification,
+                    component: P2PCashier,
+                    getTitle: () => localize('Deriv P2P'),
+                    icon_component: 'IcDp2p',
+                    is_invisible: true,
+                    is_affiliate: false,
+                },
+                {
+                    id: 'gtm-onramp-tab',
+                    path: routes.cashier_onramp,
+                    component: OnRamp,
+                    getTitle: () => localize('Fiat onramp'),
+                    icon_component: 'IcCashierOnRamp',
+                    is_affiliate: false,
+                },
+            ],
+        },
+    ];
 
-let routesConfig;
+    if (is_cra) {
+        cashier_module.forEach(cashier => {
+            if (cashier.routes) {
+                cashier.routes = cashier.routes.filter(route => route.is_affiliate);
+            }
+        });
+    }
+
+    return cashier_module;
+};
 
+let routes_config, cra_routes_config;
 // For default page route if page/path is not found, must be kept at the end of routes_config array
 const route_default = { component: Page404, getTitle: () => localize('Error 404') };
 
-const getRoutesConfig = ({ is_appstore }) => {
-    if (!routesConfig) {
-        routesConfig = initRoutesConfig({ is_appstore });
-        routesConfig.push(route_default);
+const getRoutesConfig = is_cra => {
+    if (!routes_config && !cra_routes_config) {
+        routes_config = initRoutesConfig(false);
+        routes_config.push(route_default);
+        cra_routes_config = initRoutesConfig(true);
+        cra_routes_config.push(route_default);
     }
-    return routesConfig;
+
+    return is_cra ? cra_routes_config : routes_config;
```

Comment:
@george-usynin-binary Is there any time that both `routes_config` and `cra_routes_confog` can be filled at the same time?
If no, could you rewrite the method like this:

`const getRoutesConfig = is_cra => {
    if (!routes_config) {
        routes_config = initRoutesConfig(is_cra);
        routes_config.push(route_default);
    }
    return routes_config;
};`

Reply:
@Bahar-FS the reason why we create both configs is because:
when you first load the app and there will be CRA account, cra-config will be created. it's ok until the client switches  to CR account (e.g. BTC), 
after switching we will call getRoutesConfig function, BUT as config is already created, we will not recreate the config with (is_cra = false) and after switching we can't see appropriate tabs (Deposit, Payment Agent) because we still will use CRA config. hope it was clear. So that's the main idea of the both configs. Maybe there exist another solution to resolve my concern ...

Reply:
I see @george-usynin-binary 
However, the way you wrote the condition has an issue IMO, consider the time that `is_cra=false` and `cra_routes_config` is true, so the second condition will run, which is not preferred and you've handled not to return the incorrect result in the return part, but the second condition should not even run in this condition.

Reply:
@george-usynin-binary Can’t we just return `initRoutesConfig(is_cra)` and remove those unnecessary variables and wrap it with `useCallback`? 🤔

Reply:
@farzin-fs we remove this implementation with both configs, anyway thanks for the tip!